src/vote_transaction.rs

```diff
@@ -16,7 +16,7 @@ use system_program;
 use vote_program::{self, Vote, VoteInstruction};
```

Comment:
ruh roh, this'll conflict a little with #2003.  Just FYI in case #2003 happens to land first